### ISSUE TITLE
tiny 10 iso booted[Success Image Report]: 

### ISSUE THREAD

**Issue by Asimopostin** [ventoy/Ventoy#1550]
### Official Website List

- [X] I have checked the list in official website and the image file is not listed there.

### Ventoy Version

1.0.72

### BIOS Mode

UEFI Mode

### Partition Style

GPT

### Image file name

tiny10 21H2 x64 beta 1.iso

### Image file checksum type

SHA256

### Image file checksum value

i dont think what the checksum is

### Image file download link (if applicable)

https://mega.nz/file/z08UGBLI#DY8SBrpKScnuHULKOCROILUG5syFuZ-ZVymnVXJQf6g

### Test envrionment

custom pc

### More Details?

This image file booted successfully in Ventoy. you need to bypass missing drivers if it appears or just load a driver, TINY 10 is a ISO from NTDEV for a super minimal install of windows, its reccomended for a emergency install. tiny 10 is not affilated with microsoft, this is a beta iso for x64 computers, theres another version for x86 on his channel

credits:https://www.youtube.com/c/NTDEV

**Comment by Ruzie** [comment]
IMO, almost every modded version of Windows (starting from 7 to 11) is bootable using Ventoy, as they are not modifying boot technology used by Windows but only modifying apps that are not related to the Windows noot process.

**Comment by ventoy** [comment]
Thanks.

**Comment by abduraimovabdurahmon** [comment]
url ishlamayapti

**Comment by osirisgothra** [comment]
image links are no longer valid :(

**Comment by EnduringGuerila** [comment]
UEFI, tiny10 started but then upon starting the install, it threw an error saying to check my installation media. I have used this ventoy usb plenty of times without issue, testing another tiny10 iso now... 

ok so for some reason the second tiny10 iso worked. 
tiny10 x64 23h2 MicroWin (4.2gb), MD5: 6410fd03e94f4a8781ac656e167ed096
failed

tiny10 x64 23h2 (3.6gb), MD5: 893f0df3bb42f3a4d63ed3632ac47d59
worked

hope that helps someone. No idea where I got these isos, it was 2 years ago
